frontend/src/actions/TableActions.js

```diff
@@ -345,7 +345,7 @@ export function updateGridTable(tableId, tableResponse) {
  * @method updateGridTableData
  * @summary Update grid table's rows and rebuild collapsed rows if necessary
  */
-export function updateGridTableData(tableId, rows) {
+export function updateGridTableData({ tableId, rows, changedIds }) {
```

Comment:
document `changedIds` param:
* what's it's type
* what does when it's set
* what does when it's not set

Reply:
any reason to not rename `changedIds` to `preserveCollapsedStateToRowIds` here too?